src/pages/settings/NewPassword.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {TextInput, View} from 'react-native';
+import Text from '../../components/Text';
+import withLocalize, {
+    withLocalizePropTypes,
+} from '../../components/withLocalize';
+import CONST from '../../CONST';
+import styles from '../../styles/styles';
+
+const propTypes = {
+    password: PropTypes.string.isRequired,
+    setPassword: PropTypes.func.isRequired,
+    passwordIsValid: PropTypes.func.isRequired,
+    onSubmitEditing: PropTypes.func.isRequired,
+    ...withLocalizePropTypes,
+};
+
+class NewPassword extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            confirmNewPassword: '',
+            passwordHintError: false,
```

Comment:
shouldShowPasswordHintError

Reply:
it doesn't show an error exactly, it changes the styling to be red on the hint. 
Could be slightly misleading as the hint text is always shown. What do you think?

Reply:
Good point, this is fine to leave then 👍